stable/jenkins/values.yaml

```diff
@@ -97,10 +97,14 @@ master:
   # Enable Kubernetes Liveness and Readiness Probes
   # ~ 2 minutes to allow Jenkins to restart when upgrading plugins. Set ReadinessTimeout to be shorter than LivenessTimeout.
   healthProbes: true
-  healthProbesLivenessTimeout: 90
-  healthProbesReadinessTimeout: 60
-  healthProbeReadinessPeriodSeconds: 10
-  healthProbeLivenessFailureThreshold: 12
+  healthProbesLivenessTimeout: 20
```

Comment:
`healthProbesLivenessTimeout` should be 5 as it's been previously rendered as `timeoutSeconds: 5`